### PR TITLE
[IMP] workaround for finding known_hosts file (#396)

### PR DESCRIPTION
**Pull request by bodi000**

### CODE REVIEW

**Comment by yvaucher** [comment]
@bodi000 can you detail a bit more what your intent is here?
Do you have SSL issues?

File: l10n_ch_fds_postfinance/models/fds_postfinance_account.py [hunk 1]
```diff
@@ -103,10 +103,13 @@ def verify_directories_button(self):
                 key[0].private_key_crypted)
             key_pass = self.authentication_key_ids.config()
 
+            cnopts = pysftp.CnOpts(knownhosts='/etc/ssh/ssh_known_hosts')
```
Comment (ecino) [inline l10n_ch_fds_postfinance/models/fds_postfinance_account.py]: I fear this will fail on a non-linux host.

**Review by ecino — COMMENTED** [review]
In the documentation of pysftp, changing the knownhosts is not recommended:
image
Are you sure there is no other fix for this? Normally you should connect with ssh outside odoo, with the odoo user to the server at least one time to add the host to your known hosts.

**Comment by bodi000** [comment]
Hello. Yes, as it is explained in the issue #396...
However, the behaviour comes from a bug at pysftp level. I filed an issue there and a PR(which I need to rework ;))

This is just the workaround for using a system user without HOME folder.

If the upstream bug gets solved, no need for this PR.

Thank you for your understanding
B

File: l10n_ch_fds_postfinance/models/fds_postfinance_account.py (hunk 1 above)
Comment (bodi000) [reply l10n_ch_fds_postfinance/models/fds_postfinance_account.py]: Hello. As I mentioned I use a system user. No Home folder, so no usefullness into connecting once.

Please park this for a few days, i am tryng to solve this with pysftp directly. My issue was recongnised as a bug there ..

**Pull request closed without merging**